chinese/chinese-tomato-egg.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Chinese Tomato Egg Stir-Fry</title>
+  </head>
+  <body>
+    <img
+      src="https://christieathome.com/wp-content/uploads/2020/10/Chinese-Tomato-Egg-Stir-Fry-11-300x300.jpg"
+      alt=""
+    />
+    <h1>Chinese Tomato Egg Stir-Fry</h1>
+    <p>
+      Chinese tomato egg stir-fry. Soft and silky eggs stir fried in a sweet
+      tangy tomato sauce with green onions. Ready in 15 minutes in one pan!
+    </p>
+    <table>
+      <tr>
+        <td><h3>Ingredients</h3></td>
+        <td><hr /></td>
+      </tr>
+      <tr>
+        <td align="right">4</td>
+        <td>
+          roma tomatoes
+          <font color="grey"
+            >ripened, sliced into quarters (or use tomatoes that you have on
+            hand)</font
+          >
+        </td>
+      </tr>
+      <tr>
+        <td align="right">6</td>
+        <td>eggs</td>
+      </tr>
+      <tr>
+        <td align="right">1</td>
+        <td>green onion <font color="grey">finely chopped</font></td>
+      </tr>
+      <tr>
+        <td align="right">1/4</td>
+        <td>cup water</td>
+      </tr>
+      <tr>
+        <td align="right">1</td>
+        <td>teaspoon sesame oil</td>
+      </tr>
+      <tr>
+        <td align="right">2</td>
+        <td>
+          teaspoon avocado oil
+          <font color="grey">or any neutral tasting oil</font>
+        </td>
+      </tr>
+      <tr>
+        <td><h4>Sauce:</h4></td>
+      </tr>
+      <tr>
+        <td align="right">3</td>
+        <td>tablespoon ketchup</td>
+      </tr>
+      <tr>
+        <td align="right">1/2</td>
+        <td>tablespoon cornstartch</td>
+      </tr>
+      <tr>
+        <td align="right">1</td>
+        <td>teaspoon white granulated sugar</td>
+      </tr>
+      <tr>
+        <td align="right">1/2</td>
+        <td>teaspoon ginger <font color="grey">thinly sliced</font></td>
+      </tr>
+      <tr>
+        <td align="right">1/2</td>
+        <td>teaspoon salt</td>
+      </tr>
+      <tr>
+        <td align="right">1/2</td>
+        <td>cup water</td>
+      </tr>
+      <tr>
+        <td><h3>Optional garnish:</h3></td>
+      </tr>
+      <tr>
+        <td align="right">1</td>
+        <td>green onion <font color="grey">finely chopped</font></td>
+      </tr>
+      <tr>
+        <td><br /><br /></td>
+      </tr>
+      <tr>
+        <td><h3>Instructions</h3></td>
+        <td><hr /></td>
+      </tr>
```

Comment:
These two rows with 2 line breaks and one horizontal ruler seem a bit odd. Are those necessary?